app/models/llm_api_key.rb

```diff
@@ -1,8 +1,22 @@
 class LlmApiKey < ApplicationRecord
   belongs_to :user
 
-  validates :uuid, presence: true, uniqueness: true
+  validates :uuid, uniqueness: true
   validates :llm_type, presence: true
-  validates :encrypted_api_key, presence: true
   validates :description, length: { maximum: 255 }, allow_blank: true
+
+  attr_accessor :api_key
+
+  before_validation :set_uuid, :encrypt_api_key
+
+  private
+
+  def set_uuid
+    self.uuid ||= SecureRandom.uuid
+  end
+
+  def encrypt_api_key
+    self.encrypted_api_key = ApiKeyEncrypter.new.encrypt(api_key) unless api_key.blank?
```

Comment:
```suggestion
    self.encrypted_api_key = ApiKeyEncrypter.new.encrypt(api_key) if api_key.present?
```

細かい点ですが、否定形を使わずに書ける場合は unless より if を使った方が、読み手が直感的に理解しやすくなります。